nipype/interfaces/afni/model.py

```diff
@@ -203,3 +203,109 @@ def _list_outputs(self):
 
         outputs['out_file'] = self.inputs.out_file
         return outputs
+
+
+class RemlfitInputSpec(AFNICommandInputSpec):
+    # mandatory files
+    in_files = InputMultiPath(
+        File(
+            exists=True),
+        desc='Read time series dataset',
+        argstr='-input %s',
```

Comment:
Can you do `argstr='-input "%s"'` (i.e., will it get mad about double quotes around a single file)?